macos/library/FLEFileChooserPlugin.m

```diff
@@ -30,11 +30,21 @@
 static NSString *const kPlatformMethodNameKey = @"method";
 static NSString *const kPlatformMethodArgsKey = @"args";
 
-// Method argument keys.
+// Appearance configuration keys for NSSavePanel.
+static NSString *const kPanelTitleKey = @"title";
+static NSString *const kPanelMessageKey = @"message";
+static NSString *const kOKButtonTextKey = @"prompt";
```

Comment:
This message name is very specific to NSSavePanel; these plugins are intended to be cross-platform, so should use more descriptive names. confirmButtonLabel maybe?

Reply:
I chose this key name so that its semantics communicate the parameter's association with a specific value of the FileChooserResult enum (FileChooserResult.OK) on the flutter side. That said, I'm not convinced that 'OK button text' is capturing that. So, I've tried to split the difference by changing the key to 'OKButtonLabel'.